docs/es/docs/tutorial/query-params-str-validations.md

```diff
@@ -1,170 +1,171 @@
 
-!!! info
-    Have in mind that FastAPI cares about the part:
+!!! información
+    Ten en mente que FastAPI se ocupa de la parte de:
 
     ```Python
     = None
     ```
 
-    or the:
+    o de:
 
     ```Python
     = Query(None)
     ```
 
-    and will use that `None` to detect that the query parameter is not required.
+    y usará `None` para detectar que el parámatro de consulta no es requerido.
 
-    The `Optional` part is only to allow your editor to provide better support.
+    La parte `Optional` es solo para permitirle a tu editor proveer un mejor soporte.
 
-Then, we can pass more parameters to `Query`. In this case, the `max_length` parameter that applies to strings:
+Entonces, podemos pasar más parámetros a `Query`. En este caso, el parámetro `max_length` que aplica para cadenas de texto:
 
 ```Python
 q: str = Query(None, max_length=50)
 ```
 
-This will validate the data, show a clear error when the data is not valid, and document the parameter in the OpenAPI schema *path operation*.
+Esto validará los datos, mostrará un error claro cuando los datos no sean válidos y documentará el parámetro en el esquema de OpenAPI *ruta de operación*.
 
-## Add more validations
+## Agrega más validaciones
 
-You can also add a parameter `min_length`:
+Tambien puedes agregar el parametro `min_length`:
 
 ```Python hl_lines="9"
 {!../../../docs_src/query_params_str_validations/tutorial003.py!}
 ```
 
-## Add regular expressions
+## Agrega una expresión regular
 
-You can define a <abbr title="A regular expression, regex or regexp is a sequence of characters that define a search pattern for strings.">regular expression</abbr> that the parameter should match:
+Puedes definir una <abbr title="Una expresión regular, regex o regexp es una secuencia de caracteres que define un patrón de búsqueda para cadenas de texto.">expresión regular</abbr> que el parámetro debería cumplir:
 
 ```Python hl_lines="10"
 {!../../../docs_src/query_params_str_validations/tutorial004.py!}
 ```
 
-This specific regular expression checks that the received parameter value:
+Esta expresión regular verifica que el valor del parámetro recibido:
 
-* `^`: starts with the following characters, doesn't have characters before.
-* `fixedquery`: has the exact value `fixedquery`.
-* `$`: ends there, doesn't have any more characters after `fixedquery`.
+* `^`: comienza con los siguientes caracteres, no tiene caracteres antes.
+* `fixedquery`: tiene el valor exacto de `fixedquery`.
+* `$`: termina allí, no tiene más letras después de `fixedquery`.
 
-If you feel lost with all these **"regular expression"** ideas, don't worry. They are a hard topic for many people. You can still do a lot of stuff without needing regular expressions yet.
+Si te sientes perdido con todo esto de **"expresiones regulares"**, no te preocupes. Son un tema difícil para muchas personas. Todavía puedes hacer muchas cosas sin necesidad de usar expresiones regulares todavía.
```

Comment:
++ Si te sientes perdido con todo eso de las **"expresiones regulares"**, no te preocupes. Son un tema difícil para mucha gente. Todavía puedes hacer muchas cosas sin utilizarlas.

Reply:
Este fue el cambio que realicé en esta recomendación: 
```suggestion
Si te sientes perdido con todo estos de **"expresiones regulares"**, no te preocupes. Son un tema difícil para mucha gente. Todavía puedes hacer muchas cosas sin necesitar expresiones regulares todavía.
```
dejé `estos` en lugar de `eso` porque  en ingles estan usando `these`, que seria el plural de `this`. :)